src/components/Homepage/HomePage.test.js

```diff
@@ -0,0 +1,83 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+import axios from "axios";
+import { MemoryRouter } from "react-router-dom";
+import HomePage from "./HomePage";
+import "@testing-library/jest-dom";
+
+jest.mock("axios");
+
+const MockHomePage = () => {
```

Comment:
Incorrect name for that fn, because it's NOT a mock. You're rendering a real HomePage component. That fn should be called: "renderHomePage".

Reply:
fixed